### DISCUSSION TITLE
I have a great idea...

### DISCUSSION THREAD

**Discussion by Blakedows2001** [ScratchAddons/ScratchAddons#4336] [Ideas] [answered]
Add a notifier if someone sees your projects.

**Comment by Samq64** [comment]
> Add a notifier if someone sees your projects.

I don't think that's possible and even if it was, there would be way too many messages.

**Comment by WorldLanguages** [comment, marked as answer]
Impossible

**Reply by CST1229** [reply]
Technically kinda possible if you check the view count, but that would be impractical and wouldn't let you see who saw your project.
